service/src/main/resources/HTMLTemplate.html

```diff
@@ -201,12 +205,15 @@ <h1 class="heading heading--one">
 </header>
 <main>
   <h2 class="heading heading--two u-margin-bottom-10">
-    {% if case_data.applicant is empty or case_data.applicant.name is empty %}
-      Application
-    {% else %}
-      Application from {{ case_data.applicant.name }}
+    {% if case_data.applicants is not empty %}
```

Comment:
Same here. Am I right saying that it will only fetch data from migrated cases? What about old ones?

Reply:
👍